validator_client/src/cli.rs

```diff
@@ -269,19 +269,6 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                     execution payload construction during proposals.")
                 .takes_value(false),
         )
-        .arg(
-            Arg::with_name("strict-fee-recipient")
```

Comment:
Thoughts on keeping this flag as a no-op with a warning? Or if we're doing a v3.2.0 release we can just mark it backwards incompatible and continue with deletion

Reply:
That's probably more sensible